test/integration/test_sampler.py

```diff
@@ -134,8 +134,8 @@ def test_sampler_optimization_level(self, service):
             sampler = Sampler(session=session)
             sampler.options.optimization_level = 3
             result = sampler.run(self.bell).result()
-            self.assertAlmostEqual(result.quasi_dists[0]["11"], 0.5, delta=0.05)
-            self.assertAlmostEqual(result.quasi_dists[0]["00"], 0.5, delta=0.05)
+            self.assertAlmostEqual(result.quasi_dists[0][3], 0.5, delta=0.05)
+            self.assertAlmostEqual(result.quasi_dists[0][0], 0.5, delta=0.05)
 
```

Comment:
Please also test that the `shots` and `stddev_upper_bound` properties of the Quasidistribution are set correctly.